astropy/cosmology/tests/test_cosmology.py

```diff
@@ -1081,13 +1081,13 @@ def test_critical_density():
     #  units by default; see the comment at the top of core.py
     tcos = core.FlatLambdaCDM(70.4, 0.272, Tcmb0=0.0)
     assert allclose(tcos.critical_density0,
-                    9.31000324385361e-30 * u.g / u.cm**3)
+                    9.309668456020899e-30 * u.g / u.cm**3)
```

Comment:
Ah, good that those changes are understood! It will be important to mention this in the what's new section.